### PR TITLE
[bugfix]Add logs to track sequence of events for table creation

### PR DESCRIPTION
**Pull request by aishikbh**
There are not enough logs for sub steps during table creation. Added logs to track the sequence of events to help debug issues in the future.

### CODE REVIEW

File: pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java [hunk 1]
```diff
@@ -1573,6 +1575,8 @@ public void addTable(TableConfig tableConfig)
           + "If the external view is not removed after a long time, try restarting the servers showing up in the "
           + "external view");
     }
+
+    LOGGER.info("Adding table {}: Validate table tenant config", tableNameWithType);
```
Comment (snleee) [inline pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java:1579]: Let's rephrase this as `Validate table configs`

Also, let's add one line before 1594: `Creating table config in the property store`

**Review by snleee — APPROVED** [review]
LGTM otherwise

File: pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java (hunk 1 above)
Comment (aishik94) [reply pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java]: Addressed the comments

**Review by swaminathanmanish — APPROVED** [review]
